staker-rewards/test/data/test-cache.json

```diff
@@ -0,0 +1,149 @@
+      "members": [
+        "0xd1F2A77a0dAD76Ef8B87002763134f4863C870A4",
+        "0xb038a8B2975cD5dE392683f5CD4989d3b1E75648",
+        "0x855A951162B1B93D70724484d5bdc9D00B56236B"
+      ],
+      "creationBlock": 3,
+      "creationTimestamp": 3000,
+      "status": {
+        "name": "closed",
+        "timestamp": 3100
+      }
+    },
+    {
+      "address": "0xC6f0a8726EB67910A24074e3FbB4e1aEF1671C8a",
+      "members": [
+        "0xF1De9490Bf7298b5F350cE74332Ad7cf8d5cB181",
+        "0xd1F2A77a0dAD76Ef8B87002763134f4863C870A4",
+        "0x855A951162B1B93D70724484d5bdc9D00B56236B"
+      ],
+      "creationBlock": 4,
+      "creationTimestamp": 4000,
+      "status": {
+        "name": "closed",
+        "timestamp": 4100
+      }
+    },
+    {
+      "address": "0xf876aE82E3Ef9a67ad4E9eA23eFa4de2D85DA6fb",
+      "members": [
+        "0xF1De9490Bf7298b5F350cE74332Ad7cf8d5cB181",
+        "0x855A951162B1B93D70724484d5bdc9D00B56236B",
+        "0x5CA1F949c75833432d6BC8E3cb6FB23386F63426"
+      ],
+      "creationBlock": 5,
+      "creationTimestamp": 5000,
+      "status": {
+        "name": "closed",
+        "timestamp": 5100
+      }
+    },
+    {
+      "address": "0x6A1074025E942138Ac634613deC0E5828e51820c",
+      "members": [
+        "0xF1De9490Bf7298b5F350cE74332Ad7cf8d5cB181",
+        "0x39d2aCBCD80d80080541C6eed7e9feBb8127B2Ab",
+        "0x5CA1F949c75833432d6BC8E3cb6FB23386F63426"
+      ],
+      "creationBlock": 6,
+      "creationTimestamp": 6000,
+      "status": {
+        "name": "terminated",
+        "timestamp": 6100,
+        "cause": "keygen-fail"
+      }
+    },
+    {
+      "address": "0x6EAD0C3A59B4b5258f72317Fa24c5D759886158B",
+      "members": [
+        "0x5CA1F949c75833432d6BC8E3cb6FB23386F63426",
+        "0xF1De9490Bf7298b5F350cE74332Ad7cf8d5cB181",
+        "0x9d0Cc9194EfD1F9Bf02a7AFC57Ae9769c1207Aa4"
+      ],
+      "creationBlock": 7,
+      "creationTimestamp": 7000,
+      "status": {
+        "name": "terminated",
+        "timestamp": 7100,
+        "cause": "signature-fail"
+      }
+    },
+    {
+      "address": "0x060341C3fAE3929B96Afcc1449Ac51D34A3639f8",
+      "members": [
+        "0xF1De9490Bf7298b5F350cE74332Ad7cf8d5cB181",
+        "0x39d2aCBCD80d80080541C6eed7e9feBb8127B2Ab",
+        "0x186E82df0a09534537d0D8680D03b548628ab288"
+      ],
+      "creationBlock": 8,
+      "creationTimestamp": 8000,
+      "status": {
+        "name": "terminated",
+        "timestamp": 8100,
+        "cause": "other"
+      }
+    },
+    {
+      "address": "0x475666ECE72d4e53Fa2F1EDEc21564e0dBFf3665",
+      "members": [
+        "0xF1De9490Bf7298b5F350cE74332Ad7cf8d5cB181",
+        "0x88Ed51f84c21Ae246c23137D090cdF441009D916",
+        "0xb038a8B2975cD5dE392683f5CD4989d3b1E75648"
+      ],
+      "creationBlock": 9,
+      "creationTimestamp": 9000,
+      "status": {
+        "name": "active",
+        "timestamp": 9000
+      }
+    },
```

Comment:
Can we add one more keep here where `0xF1De9490Bf7298b5F350cE74332Ad7cf8d5cB181` is not a member? I see we already have one on 3rd position but just one more at the end would make me feel more comfortable.

Reply:
Ok done